vividus-extension-selenium/src/main/resources/steps/defaults/deprecated.steps

```diff
@@ -1,17 +1,17 @@
 Composite: When I wait until element located `$locator` appears
-!-- WARNING: The step "When I wait until element located `$locator` appears" is deprecated and will be removed in VIVIDUS 0.6.0
+!-- VersionToRemoveStep: "0.6.0" ReplacementFormatPattern: "When I wait until element located by `%1$s` appears"
```

Comment:
RemovalVersion: ReplacementPattern:
isn't it more friendly?

Reply:
> RemovalVersion: ReplacementPattern: isn't it more friendly?

Yep
But I tried to match the values of the `Replacement` annotation to keep everything consistent
Need to discuss this moment